app/bundles/CoreBundle/Helper/DateTimeHelper.php

```diff
@@ -13,34 +13,16 @@
 
 class DateTimeHelper
 {
-    /**
-     * @var string
-     */
     private $string;
```

Comment:
May I ask why the types are removed here? Not sure why the CS fixer would do that... 😕

Reply:
@escopecz do you have idea why CS fixer remove types?

Reply:
CS Fixed doesn't do it for class properties for me nor any other PR I've seen so far. Can you try to revert and push again?

Reply:
@escopecz Ok, reverted. I will try investigate it why this happen